src/gfx/math.hpp

```diff
@@ -40,6 +40,11 @@ template <size_t Alignment> inline constexpr size_t alignTo(size_t size) {
   }
 }
 
+template <typename T> inline bool isRoughlyEqual(T val, T target, T tolerance = 0.05f) {
+  T delta = val - target;
+  return delta > -tolerance && delta < tolerance;
```

Comment:
Epsilon is preferred esp. since you are already using delta.

Reply:
Not sure if you are talking about renaming `tolerance`or `delta. Most likely `tolerance`.

I would agree if the value used was very small and indeed an epsilon (i.e. a value that adding or subtracting doesn't produce a noticeable difference). but I think here it is a rough comparison which might have noticeable difference, so IMO `tolerance` is more appropriate.

Reply:
That was my reasoning for using tolerance yes ^